api/src/main/java/me/lokka30/treasury/api/economy/transaction/EconomyTransaction.java

```diff
@@ -60,7 +57,8 @@ public EconomyTransaction(
             @Nullable Temporal timestamp,
             @NotNull EconomyTransactionType economyTransactionType,
             @Nullable String reason,
-            final BigDecimal transactionAmount
+            final BigDecimal transactionAmount,
```

Comment:
since we're annotating bigdecimals with `@notnull`, maybe we should have it here too?

Reply:
Great point. I'm unsure why they weren't annotated in the first place, was there any known reason for it?

Reply:
because you can't annotate a primitive with `@notnull` and probably creatorfromhell used replaceAll feature of his IDE to replace them